kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -216,6 +169,23 @@
       closeCreateUserModal() {
         this.creatingUser = false;
       },
+      matchesText(user) {
```

Comment:
Was originally reluctant to put this in `methods` because it's not used anywhere but `visibleUsers()`. I tried defining them as functions inside the computed property, but `this` isn't defined there.

@indirectlylit thoughts on this? Are there any consequences to including these in `methods` like this? I know you did it this way in the comment you posted originally, just wanted to know your opinion.